cmd/auth/login.go

```diff
@@ -56,6 +60,38 @@ var loginCmd = &cobra.Command{
 			return err
 		}
 
+		if configureCluster {
+			err := root.MustWorkspaceClient(cmd, args)
```

Comment:
This won't work -- it will try to load the profile from the profile flag. Instead it should use the config that was synthesized above and saved to a profile to access the workspace.

Btw, also note that the configure cluster flag will be incompatible with the account ID flag; clusters are not applicable at the account level.